MODULE.bazel

```diff
@@ -6,12 +6,6 @@ module(
     compatibility_level = 1,
 )
 
-cypress = use_extension("//cypress:extensions.bzl", "cypress")
```

Comment:
We want to punt the toolchain registration to user space so they can pick their cypress version right?

Reply:
Actually, that reminds me that we don't have the MODULE.bazel snipped in our release notes: 

https://github.com/aspect-build/rules_cypress/releases/tag/v0.4.0

like we do for some of other rulesets. Its a mixed bag

bzlmod snippets: rules_esbuild, rules_js, rules_swc, rules_ts
no bzlmod snippets: rules_cypress, rules_jasmine, rules_jest, rules_rollup, rules_terser, rules_webpack

Reply:
Bingo, that was an oversite on my part. Its moved now. Lets add the bzlmod snippet either manually or in a seperate PR

Reply:
Yeah, bzlmod snipped can be in a follow-up 👍 This is not the only rule set that is missing one.